tbc/stats.go

```diff
@@ -4,7 +4,7 @@ import (
 	"strconv"
 )
 
-type Stats []float64
+type Stats [StatLen]float64
```

Comment:
Love this - I had this on my local branch for simplifying somethings and appreciate this :)

Reply:
;)